docs/explanations/audit-scanner.md

```diff
@@ -0,0 +1,267 @@
+
+See [here](../howtos/audit-scanner) how to enable Audit Scanner.
+
+## Audit intrinsic limitations
+
+### Policies checking `UPDATE` events
+
+Policies can inspect `CREATE`, `UPDATE`, and `DELETE` events. The audit check
+feature cannot intrinsically test for `UPDATE` events, as it doesn't know
+exactly which part of the resource needs to `UPDATE` in a meaningful way.
+
+Policies interested in `UPDATE` events will be run by the audit check, and
+exercised for other operations, but not the `UPDATE` operation.
+
+### Policies checking user and user group information
+
+Each Kubernetes admission request object contains information about which user
+(or ServiceAccount) initiated the event, and to which group they belong.
+
+The audit feature cannot possibly infer what user or ServiceAccount to test
+against, and will not produce a meaningful evaluation, as it will use its own
+ServiceAccount. For these cases, the policy should be configured to be skipped
+from audit checks.
+
+### Policies relying on external data
+
+Policies can request and use external data when performing an evaluation. These
+policies can be evaluated by the audit checks, but their outcome can change over
+time depending on the external data.
+
+### Policies targeting `*`
+
+Policies targeting any kind of Kubernetes resources are not being evaluated for
+now. Users have to be explicit about the Kubernetes resources a policy targets
+for now.
+
+
+## Policy Reports
+
+When using the Kubewarden Audit Scanner, the results of the policy scans are
+stored in [PolicyReport](https://htmlpreview.github.io/?https://github.com/kubernetes-sigs/wg-policy-prototypes/blob/master/policy-report/docs/index.html) Custom Resource Definitions (CRDs).
+
+:::caution
+Note that the PolicyReport CRDs are under development in the `wg-policy`
+Kubernetes group. Therefore, this documentation can be out of date if a new
+version of the CRDs is released.
+
+Check the `wg-policy` group
+[repository](https://github.com/kubernetes-sigs/wg-policy-prototypes) out for
+more information about the CRDs.
+:::
+
+These CRDs provide a structured way to store and manage the audit results. Each
+namespace scanned by the audit scanner will have a separate namespaced
+`PolicyReport` associated with it. Cluster wide resources have a separate
+`ClusterPolicyReport` as well.
+
+To use the audit scanner, you need to manually install the PolicyReport CRDs
+or use the version installed by `kubewarden-crds`.  These CRDs define the
+structure and schema for the audit reports generated by the scanner.
+
+The audit results generated by the scanner includes various information, such as
+the policy that was evaluated, the resource being scanned, the result of the
+evaluation (pass, fail, or skip), and a timestamp indicating when the
+evaluation took place. Additionally, you can optionally define severity and
+category annotations for your policies in the `metadata.yml` file.
```

Comment:
I think we should not mention `metadata.yml` file. That's only for policy authors, not policy consumers.

```suggestion
category annotations for your policies.
```